contracts/smart-account/modules/SecurityPolicyManagerPlugin.sol

```diff
@@ -87,8 +87,10 @@ contract SecurityPolicyManagerPlugin is ISecurityPolicyManagerPlugin {
 
             switch success
             case 0x1 {
-                moduleInstallationSuccess := mload(ptr)
-                module := mload(add(ptr, 0x60))
+                if eq(size, 0x80) {
```

Comment:
let success := call(
                gas(),
                caller(),
                0,
                savePtr,
                sub(ptr, savePtr),
                0,
                0 **// provide expected return size here**
            )
            
            Should't this be enough?

Reply:
So the issue here is that in a situation where the setupContract doesn't return the module address after setupCalldata is executed, then the returndatasize of this call would just be 0x20 (success).
If it does return then the length of 0x80 (success + offset + length + module address).
So even if we hard-coded 0x80 in the call, we'll still have to manually check if 0x80 bytes were actually returned by calling returndatasize